less/home-page.less

```diff
@@ -27,6 +67,11 @@
     padding-bottom: 30px;
     border-bottom: 2px dashed @navigation-border;
 
+    @media @mobile {
+      padding-bottom: 0;
+      flex-direction: column-reverse;
```

Comment:
really good usage of column reverse and adding the .inverse-content class to reposition the elements and images in mobile.